lib/util/support.js

```diff
@@ -33,6 +42,16 @@ function getParenthesizedSubstrings(string) {
   return substrings;
 }
 
+/**
+ * Check whether the current browser is a mobile browser
+ * @param {string} userAgent
+ * @returns {boolean}
+ */
+function isMobile(userAgent) {
+  userAgent = userAgent || navigator.userAgent;
+  return /Mobi/.test(userAgent);
```

Comment:
Any reason why we are using `Mobi` instead of `Mobile`?

Reply:
Best practice, as mentioned [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent). Scroll all the way to the bottom. But to summarize, all browsers uses `Mobile`. But opera/presto uses `Mobi`